jupyter_sphinx/__init__.py

```diff
@@ -272,8 +272,9 @@ def setup(app):
     app.add_directive("jupyter-execute", JupyterCell)
     app.add_directive("jupyter-kernel", JupyterKernel)
     app.add_directive("thebe-button", ThebeButton)
-    app.add_role("jupyter-download:notebook", jupyter_download_role)
-    app.add_role("jupyter-download:script", jupyter_download_role)
+    app.add_role("jupyter-download:notebook", JupyterDownloadRole())
```

Comment:
I notice you are passing instances and not classes here, unlike in the surrounding code. Is there a reason for that?

Reply:
its just the way `ReferenceRole` works, this how it's used in sphinx itself

Reply:
OK, I see. Sphinx API requires adding a class for directives, and a function for roles (which in this case is a class instance, it seems).

Reply:
when it requires a `Callable` for the role; so either a function, or a class instance with a `__call__` method